docs/machine-learning/tutorials/iris-clustering.md

```diff
@@ -0,0 +1,207 @@
+
+Note that we specify that the data set should be split in three clusters.
+
+## Train the model
+
+The steps added in the preceding sections prepared the pipeline for training. None of the steps has been executed. The `pipeline.Train<TInput, TOutput>` method produces the model that takes in an instance of the `TInput` type and outputs an instance of the `TOutput` type. Add the following code into the `Train` method:
+
+[!code-csharp[Train the model and return](../../../samples/machine-learning/tutorials/IrisClustering/Program.cs#9)]
+
+### Save the model
+
+Before you go onto the next step, save the model to a .zip file. To do that add the following code to the `Main` method below the call to the `Train` method:
+
+[!code-csharp[Save the model](../../../samples/machine-learning/tutorials/IrisClustering/Program.cs#10)]
+
+Using `await` in the `Main` method means the `Main` method must have the `async` modifier and return a `Task`:
+
+[!code-csharp[Make the Main method async](../../../samples/machine-learning/tutorials/IrisClustering/Program.cs#11)]
+
+You also need to add the following `using` directive at the top of the *Program.cs* file:
+
+[!code-csharp[Add System.Threading.Tasks using](../../../samples/machine-learning/tutorials/IrisClustering/Program.cs#12)]
+
+Because the `async Main` method is the feature added in C# 7.1 and the default language version of the project is C# 7.0, you need to change the language version to C# 7.1 or higher. To do that, right-click the project node in **Solution Explorer** and select **Properties**. Select the **Build** tab and select the **Advanced** button. In the dropdown, select  **C# 7.1** (or a higher version). Select the **OK** button.
+
+## Use the model for predictions
+
+Create the `TestIrisData` class to house test data instances:
+
+1. In **Solution Explorer**, right-click the project, and then select **Add** > **New Item**.
+1. In the **Add New Item** dialog box, select **Class** and change the **Name** field to *TestIrisData.cs*. Then, select the **Add** button.
+1. Modify the class to be static like in the following example:
+
+   [!code-csharp[Make class static](../../../samples/machine-learning/tutorials/IrisClustering/TestIrisData.cs#1)]
+
+This tutorial introduces one iris data instance within this class. You can add other scenarios to experiment with the model. Add the following code into the `TestIrisData` class:
+
+[!code-csharp[Test data](../../../samples/machine-learning/tutorials/IrisClustering/TestIrisData.cs#2)]
+
+To find out the cluster to which the specified item belongs to, go back to the *Program.cs* file and add the following code into the `Main` method:
+
+[!code-csharp[Predict and output results](../../../samples/machine-learning/tutorials/IrisClustering/Program.cs#13)]
+
+Run the program to see which cluster contains the specified data instance and squared distances from that instance to the cluster centroids.
```

Comment:
Please show the output like this [example in the Sentiment Analysis tutorial](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis#results).